tasks/app/views/tasks/index.html.erb

```diff
@@ -10,7 +10,9 @@
       <% if gray %>
         <div class="gray-box">
           <span><%= num %>    </span><a href="/tasks/<%= task.id %>"><%= task.name.upcase %></a>
-          <%= button_to "delete", "/tasks/#{task.id}", method: :delete, data: { :confirm => "Are you sure?" }, class: "delete-button" %>
```

Comment:
All of these `button_to` lines are duplicated exactly the same for each of the conditions checked in the loop (`task.complete.nil?` and `gray`). How might we DRY this out to only include the button code once?